plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileGenericPage.java

```diff
@@ -10,125 +10,64 @@
  ******************************************************************************/ 
 package org.jboss.reddeer.eclipse.datatools.ui.wizard;
 
-import org.jboss.reddeer.swt.impl.button.CheckBox;
-import org.jboss.reddeer.swt.impl.text.LabeledText;
-
 /**
  * A wizard page for creating new Generic JDBC (H2,..) profile.
  * 
  * 
- * @author Jiri Peterka
+ * @author Jiri Peterka, mlabuda@redhat.com
  * 
  */
 public class ConnectionProfileGenericPage extends ConnectionProfileDatabasePage {
```

Comment:
name & package is wrong. Should be eclipse.datatools.connectivity.db.generic.internal.ui.GenericJDBCDBProfileDetailsWizardPage